.github/workflows/release.yaml

```diff
@@ -0,0 +1,90 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract version from tag
+        id: extract_version
+        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
+
+      - name: Debug version
+        run: echo ${{ env.VERSION }}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login Helm to DockerHub registry
+        run: |
+           helm registry login registry-1.docker.io -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin <<< "${{ secrets.DOCKERHUB_TOKEN }}"
+
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Build and publish Docker image (init container)
+        env:
+          VERSION: ${{ env.VERSION }}
+        run: |
+          echo "Building Docker init image with version: ${VERSION}"
+          docker build -t kubeshop/monokle-admission-controller-init:latest -t kubeshop/monokle-admission-controller-init:${VERSION} ./admission-controller/init/
+          docker push kubeshop/monokle-admission-controller-init:${VERSION}
+          docker push kubeshop/monokle-admission-controller-init:latest
+
+      - name: Build and publish Docker image (server container)
+        env:
+          VERSION: ${{ env.VERSION }}
+        run: |
+          echo "Building Docker server image with version: ${VERSION}"
+          docker build -t kubeshop/monokle-admission-controller-server:latest -t kubeshop/monokle-admission-controller-server:${VERSION} ./admission-controller/server/
```

Comment:
Is there a difference between `kubeshop/monokle-admission-controller-server` & `kubeshop/monokle-admission-controller`?

Reply:
`kubeshop/monokle-admission-controller` is a Helm chart while `kubeshop/monokle-admission-controller-server` is a Docker image for webhook server.